DNN Platform/Library/Services/Installer/Writers/ModulePackageWriter.cs

```diff
@@ -154,7 +154,7 @@ private static void ProcessControls(XPathNavigator controlNav, string moduleFold
 
             //Write controlSrc
             string controlSrc = Util.ReadElement(controlNav, "src");
-            if (!(controlSrc.ToLower().StartsWith("desktopmodules") || !controlSrc.ToLower().EndsWith(".ascx")))
+            if (!(controlSrc.ToLowerInvariant().StartsWith("desktopmodules") || !controlSrc.ToLowerInvariant().EndsWith(".ascx")))
```

Comment:
Please use `String#StartsWith(String, StringComparison)` and `String#EndsWith(String, StringComparison)`